tomls/omnibus-base-mainnet-andromeda/spot/usdc.toml

```diff
@@ -1,5 +1,5 @@
 [setting.synthUsdcMaxMarketCollateral]
-defaultValue = "<%= parseEther(String(50_000)) %>"
+defaultValue = "<%= parseEther(String(1_000_000)) %>" #$1m USDC LP cap
```

Comment:
Need to update the e2e test file as well (for mainnet)

Reply:
https://github.com/Synthetixio/SIPs/pull/1807